integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDomainInPV.java

```diff
@@ -135,7 +139,7 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
     labelMap.put("istio-injection", "enabled");
 
     assertDoesNotThrow(() -> addLabelsToNamespace(domainNamespace,labelMap));
-    assertDoesNotThrow(() -> addLabelsToNamespace(opNamespace,labelMap));
+    // assertDoesNotThrow(() -> addLabelsToNamespace(opNamespace,labelMap));
```

Comment:
Comment can be removed.

Reply:
I should uncomment the statement. Istio should be enabled in Operator Namespace.